modules/multi/hash_demulti/demuxem.nf

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env nextflow
 nextflow.enable.dsl = 2
 
-process demuxem {
-    publishDir "$projectDir/$params.outdir/$sampleId/$params.mode/hash_demulti/demuxem", mode:'copy'
+process demuxem{
+    publishDir "$params.outdir/$sampleId/$params.mode/hash_demulti/demuxem", mode:'copy'
     label 'small_mem'
 
-    conda 'bioconda::pegasuspy bioconda::scanpy bioconda::demuxEM'
-
+    conda "bioconda::pegasuspy demuxEM scanpy" 
```

Comment:
scanpy should be taken from conda-forge

Reply:
ahh, good point. maybe this causes my issues.